addons/func_godot/src/netradiant_custom/netradiant_custom_gamepack_config.gd

```diff
@@ -208,10 +214,15 @@ func do_export_file() -> void:
 	print("Exporting NetRadiant Custom Default Buld Menu to ", target_file_path)
 	file = FileAccess.open(target_file_path, FileAccess.WRITE)
 	if file != null:
-		file.store_string("<?xml version=\"1.0\"?><project version=\"2.0\"></project>")
-		file.close()
-	else:
-		printerr("Error: Could not modify " + target_file_path)
+		file.store_string("<?xml version=\"1.0\"?>\n<project version=\"2.0\">\n")
+		
+		for key in build_xml_variables.keys():
+			file.store_string('\t<var name="%s">%s</var>\n' % [key, build_xml_variables[key]])
+			
+		for key in build_xml_commands.keys():
+			file.store_string('\t<build name="%s">\n\t\t<command>%s</command>\n\t</build>\n' % [key, build_xml_commands[key]])
```

Comment:
This should be a for loop that loops through the build commands array, since you can have multiple commands in one build option. Even though we haven't found a use for multiple in our own projects, we should still leave the theoretical possibility open since it's easy enough to implement it.

Better still would be an option to support both, since sometimes you only want the one option:
```gdscript
	for key in default_build_menu_commands.keys():
		file.store_string("'\t<build name=" + key + ">\n")
		if default_build_menu_commands[key] is String:
			file.store_string("\t\t<command>" + default_build_commands[key] + "</command>\n"
		elif default_build_menu_commands[key] is Array:
			var commands: Array = default_build_menu_commands[key]
			for command in commands:
				if command is String:
					file.store_string("\t\t<command>" + (command as String) + "</command>\n"
				else:
					push_error("ERROR: A command in build option \"" + key + "\" commands Array is not a String")
		else:
			push_error("ERROR: Build option \"" + key + "\" command is not a String or Array of Strings")
```